docs/topics/serialization.txt

```diff
@@ -307,6 +311,26 @@ The JSON serializer uses ``DjangoJSONEncoder`` for encoding. A subclass of
 
 .. _ecma-262: https://www.ecma-international.org/ecma-262/5.1/#sec-15.9.1.15
 
+.. _serialization-formats-jsonl:
+
+JSONL
+-----
+
+.. versionadded:: 3.2
+
+JSONL serializer output looks very close to JSON, The only difference is each
+object is stored in a single line, objects separated by new lines and
+each line contains a valid JSON object, using same sample data,
+serialized data look like this::
+
+    { "pk": "4b678b301dfd8a4e0dad910de3ae245b", "model": "sessions.session", "fields": { "expire_date": "2013-01-16T08:16:59.844Z", ... }}
+
+The benefit of JSONL serializer compared to JSON is the serialized data can be
+processed line by line (instead of loading whole data at once, which causes
+memory error on large dumps) therefore it's recommended to use JSONL format for
+:djadmin:`dumpdata` for large databases to avoid memory error when you want to
+:djadmin:`loaddata` later.
```

Comment:
```suggestion
JSONL can be useful for populating large databases, since the data can be
processed line by line, rather than being loaded into memory all at once. 
```